### ISSUE TITLE
Fix `declaration-property-value-no-unknown` false positive for `aspect-ratio`

### ISSUE THREAD

**Issue by pamelalozano16** [stylelint/stylelint#8027]
### What minimal example or steps are needed to reproduce the bug?

From https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio#syntax

```scss
img  {
  aspect-ratio: auto 3/4;
}
```

### What minimal configuration is needed to reproduce the bug?

```
{
  "extends": [
    "stylelint-config-standard"
  ],
  "rules": {
    "declaration-property-value-no-unknown": true
  }
}
```

### How did you run Stylelint?

[Demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICWBbA5gAmAHQHbYBcBDAZwAcYwCBaAJyIKQngwGYB6AFgyIFcCIAbjyFSFKnQZMWfAWy7DcAXxAAaEADMkAGxgA5IijiIYAD0NldAOjAkSa8BFxa0CEDlwYM2EGYIxcKBIfFgBtPC8vHxICAE9dbSRcGkhnJDRqGKJAolooHwiMAF1VQp9aXl1ghCxCqJBYMG1cqSdqMloIClo46gA3Im1eGGpcCGpeXABrMYB3fBqCCphCpTwVdVSXADEIWhQGNwArEicHWDJ7RA96mPiYROSQ7xBm-xifUs8Xu4SklKcLkyxByeWePjeMA+IHWICUQA)

### Which Stylelint-related dependencies are you using?

```
{
  "stylelint": "latest",
  "stylelint-config-standard": "latest"
}
```

### What did you expect to happen?

It shouldn't raise an unexpected value error.

### What actually happened?

This generates a false positive: 
```
Unexpected unknown value "auto" for property "aspect-ratio"
[declaration-property-value-no-unknown](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown)
```

### Do you have a proposal to fix the bug?

I tried locally updating `mdn-data`, but that did not fix the issue.

**Comment by Mouvedia** [comment]
`css-tree` uses `auto | <ratio>` which is incorrect.

**Comment by ybiquitous** [comment]
@pamelalozano16 Thanks for the report with the reproducible demo. 👍🏼 

As @Mouvedia indicated, I think this is a bug of `css-tree`, which is internally used by the rule:
https://github.com/stylelint/stylelint/blob/baf7c0e4448e6e1b1d472b53df8a3dd35291449a/lib/rules/declaration-property-value-no-unknown/index.mjs#L1

See also the [`css-tree` website](https://csstree.github.io/docs/syntax/#Property:aspect-ratio):

![Image](https://github.com/user-attachments/assets/5322866f-75a9-4019-88f5-772fd11cf848)

As a workaround, you can use the [`propertiesSyntax` secondary option](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#propertiessyntax--property-syntax-) like [this demo](https://stylelint.io/demo/#N4Igxg9gJgpiBcICWBbA5gAmAHQHbYBcBDAZwAcYwCBaAJyIKQngwGYB6AFgyIFcCIAbjyFSFKnQZMWfAWy7DcAXxAAaEADMkAGxgA5IijiIYAD0NldAOjAkSa8BFxa0CEDlwYM2EGYIxcKBIfFgBtPC8vHxICAE9dbSRcGkhnJDRqGKJAolooHwiMAF1VQp9aXl1ghCxCqJBYMG1cqSdqMloIClo46gA3Im1eGGpcCGpeXABrMYB3fBrwz0ivAgqYUuWVjxWVnw6umB6kGBIAZVjkolMQ2q3d+rFKGnpGCFufWQgMAB8fjAAPK8mAA+Ar3FZKOqRKH3IqFWEqdSpFwAMQgtBQDDcACsSE4HLAyPZEDtvCAYvEYIlkh8QM1-DEfJt6pSEkkUk4XJliDk8nSGacCOCVEogA):

```json
{
  "rules": {
    "declaration-property-value-no-unknown": [
      true,
      {
        "propertiesSyntax": {
          "aspect-ratio": "auto || <ratio>"
        }
      }
    ]
  }
}
```

**Comment by ybiquitous** [comment]
So, can you please report this bug to `css-tree`? If `css-tree` fixes it, Stylelint doesn't need any changes. This bug should be resolved upstream.

**Comment by ryo-manba** [comment]
I've opened an issue to upstream.
https://github.com/csstree/csstree/issues/293

**Comment by ryo-manba** [comment]
I've submitted a PR to `mdn/data` to address this.
Once it's merged and reflected in `css-tree`, this issue should be resolved.
https://github.com/mdn/data/pull/776

**Comment by jeddy3** [comment]
This is fixed as it was addressed up stream.
